frontend/src/modules/controllers/ValueModal.tsx

```diff
@@ -30,10 +34,34 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
 }));
 
+/**
+ * @typedef SettingAdjustProps
+ *
+ * Interface for the adjusting props settings
+ *
+ * @prop {number} committedSetting updated value
+ */
 export interface SettingAdjustProps {
   committedSetting: number;
 }
 
+/**
+ * @deprecated
```

Comment:
This is listed as deprecated, but it's used by `ValueModal`, which is not listed as deprecated. Should `ValueModal` be deprecated too?

Reply:
Yes we need to deprecate that as well. Along with `ValueModal` , `ControlInfo` can also be deprecated if `HFNCGraphView` can be deprecated